### DISCUSSION TITLE
Help with GitHub for Managing Design Files

### DISCUSSION THREAD

**Discussion by Alber-S** [community/community#145368] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello GitHub Community,

I'm a graphic designer at https://custombutcherpaper.com/ where we design custom packaging like butcher paper, wax paper and parchment paper.

I'm using GitHub to manage design files but I'm running into a few issues:

**File Size:** My design files (Illustrator, Photoshop) are large and I often hit GitHub's file size limits.
**Versioning:** Keeping track of design revisions and changes is becoming hard.
**Collaboration:** Sometimes, my team struggles when multiple people need to work on the same design files.

Has anyone dealt with these problems? Any advice on how to manage large design files and track revisions on GitHub, or suggestions on tools like Git LFS?

Thanks for your help!

**Comment by kivanc57** [comment]
Hello, these are very common issues. Although, I am not in that field, I might suggest a thing or two:
**File Size**: Try to create a worse resolutioned one for the demonstration purposes.
**Versioning**: I believe it might be easy to track once you initialize the git. From then, it will track any time when you make a change.
**Collaboration**: If you use Illustrator or Photoshop, try to find a plugin or some extension which allows you to do that. I know some but they are on code editors so I am not sure that might be your case.
